### DISCUSSION TITLE
Mysterium True but cant see the nodes

### DISCUSSION THREAD

**Discussion by jokinvipo** [engageub/InternetIncome#115] [Q&A]
Hello,

I have enabled Mysterium and he gave me a output of a list with http://127.0.0.1:PORT.

How can i add this to my account? im using proxies 

Thanks alot

**Comment by engageub** [comment]
Hi,
Please access the URL in the same browser where you have logged into https://mystnodes.com/ 
Then follow the instructions mentioned in the URL.
Please note that proxies are not supported at the moment for mysterium. Only Direct Connection, VPN and Multi IP are supported.

Please see [FAQ](https://github.com/engageub/InternetIncome?tab=readme-ov-file#-why-is-mysterium-node-not-working) for more details. 



Thank you
